### ISSUE TITLE
Feature request: command line flag to hide notification title "MPD Notification"

### ISSUE THREAD

**Issue by sfgasdfsafggfgg** [eworm-de/mpd-notification#10]
I prefer it without a title, so currently I set the title to "" by changing line 19 in config.def.h to
```#define TEXT_TOPIC		""```
but it would be nice to be able to disable it via the command line without having to recompile it. I don't know C though so I can't add this.

**Comment by eworm-de** [comment]
This does not work for me. When I define `TEXT_TOPIC` to `""` (empty string) or `NULL` I get:
```
(process:31928): libnotify-CRITICAL **: notify_notification_update: assertion 'summary != NULL && *summary != '\0'' failed
```
Possibly this was added with later versions of `libnotify`, but notifications without topic are not supposed to work.

**Comment by Roboron3042** [comment]
I tried changing the default topic to show song title, but formatting was all over the place (not only <b>, but also quotes). And I don't have the slightest idea how to fix it.
![imagen](https://user-images.githubusercontent.com/22756082/112005540-2709c500-8b23-11eb-97a1-17c23b897051.png)

So, instead I went with @Ashe suggestion and just replaced the default topic with the state only. Not what I'd like, but still better than before.

![imagen](https://user-images.githubusercontent.com/22756082/112005114-c084a700-8b22-11eb-9000-e2ebc1a3dc1c.png)

I'll open a PR.
